plugins/kafka-info/src/components/KafkaInfoComponent/KafkaInfoComponent.tsx

```diff
@@ -37,10 +58,15 @@ export function KafkaInfoComponent() {
   // Get defined consumer group from entity
   const consumerGroup = entity.metadata.annotations?.[KAFKA_INFO_ANNOTATION].split(',') ?? '';
 
-  useEffect(() => {
+  const fetchLags = (proxy, token) => {
     setLoading(true);
     // Directly query a prometheus endpoint for metric data
-    fetch(`${backendUrl}/api/proxy/kafka-lag/query?query=aws_kafka_max_offset_lag_sum`)
+    fetch(`${backendUrl}/api/proxy/${proxy}/query?query=aws_kafka_sum_offset_lag_sum`, {
+      method: "GET", 
+      headers: {"Authorization": `Bearer ${token}`, 
```

Comment:
As per my comment about the tokens - this should be handled on the back end.

See https://github.com/RedHatInsights/backstage-plugin-visual-qontract/blob/b82c3f02659b85f9524ca717782ee415f0282403/app-config.yaml#L61